docs/sources/tutorials/alerting-get-started/alerting-get-started.md

```diff
@@ -0,0 +1,157 @@
+
+You will need to download the files to your local machine.
+
+
+1. Clone the [tutorial environment repository](www.github.com/grafana/tutorial-environment).
+
+    ```
+    git clone https://github.com/grafana/tutorial-environment.git
+    ```
+
+1. Change to the directory where you cloned this repository:
+
+    ```
+    cd tutorial-environment
+    ```
+
+1. Make sure Docker is running:
+
+    ```
+    docker --version
+    ```
+
+    This command will display the installed Docker version if the installation was successful. 
+1. Start the sample application:
+
+    ```
+    docker compose up -d
+    ```
+
+    The first time you run docker `compose up -d`, Docker downloads all the necessary resources for the tutorial. This might take a few minutes, depending on your internet connection.
+
+    {{< admonition type="note" >}}
+    If you already have Grafana, Loki, or Prometheus running on your system, then you might see errors because the Docker image is trying to use ports that your local installations are already using. Stop the services, then run the command again.
+    {{< /admonition >}}
+
+1. Ensure all services are up-and-running:
+
+    ```
+    docker compose ps
+    ```
+
+    In the State column, it should say Up for all services.
+
+1. Finally, the Grafana News app should be live on [localhost:8081](http://localhost:8081/).
+
+## Generate some data
+
+### Grafana News
+
+Grafana News is a straightforward application created to demonstrate the observation of data using the Grafana stack. It achieves this by generating web traffic through activities such as posting links and voting for your preferred ones
+
+To add a link:
+
+1. Enter a Title
+1. Enter a URL
+1. Click **Submit** to add the link. The link will appear listed under the Grafana News heading.
+To vote for a link, click the triangle icon next to the name of the link.
+
+## Log in to Grafana
+
+Besides being an open-source observability tool, Grafana has its own built-in alerting service.   This means that you can receive notifications whenever there is an event of interest in your data, and even see these events graphed in your visualizations.
+
+In your browser, simply navigate to [localhost:3000](http://localhost:3000). You should get logged in automatically
+
+## Create an Alert
+
+Next, we'll establish an alert within Grafana Alerting to notify us whenever our sample app experiences a specific volume of requests.
+
+In Grafana, toggle the menu at the top left side of the screen, and **navigate to Alerting** > **Alert rules**. Click on  **+ New alert rule**.
+
+## Enter alert rule name
+
+1. Make it short and descriptive as this will appear in your alert notification. For instance, server-requests-duration
```

Comment:
```suggestion
Enter an alert rule name that is short and descriptive as this will appear in your alert notification. For example, server-requests-duration.
```